src/BlockTypes/ProductCategories.php

```diff
@@ -109,9 +121,31 @@ protected function get_container_classes( $attributes = array() ) {
 			$classes[] = 'is-list';
 		}
 
-		return implode( ' ', $classes );
+		$line_height_class_and_style = AttributesUtils::get_line_height_class_and_style( $attributes );
+		$text_color_class_and_style  = AttributesUtils::get_text_color_class_and_style( $attributes );
+		$font_size_class_and_style   = AttributesUtils::get_font_size_class_and_style( $attributes );
+
+		$classes = array_filter(
+			[ $line_height_class_and_style['class'], $text_color_class_and_style['class'], $font_size_class_and_style['class'] ],
```

Comment:
`$line_height_class_and_style` might be null (according to [this line](https://github.com/woocommerce/woocommerce-gutenberg-products-block/blob/7655afcc8a5b4867698cc7d7deb8e8985e3b949b/src/Utils/AttributesUtils.php#L108-L110)), so here you will also need to check if `class` exists before using it.

Same for the other objects.